zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/NewSessionsServiceImpl.java

```diff
@@ -0,0 +1,139 @@
+package com.github.onsdigital.zebedee.session.service;
+
+import com.github.onsdigital.zebedee.reader.util.RequestUtils;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.user.model.User;
+import com.session.service.client.SessionClient;
+import com.session.service.client.SessionClientImpl;
+import com.session.service.entities.SessionCreated;
+import com.session.service.error.SessionClientException;
+import org.apache.commons.lang.StringUtils;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.Date;
+
+public class NewSessionsServiceImpl implements Sessions {
+
+    private SessionClient client;
+
+    public NewSessionsServiceImpl(SessionClient sessionClient) {
+        this.client = sessionClient;
+    }
+
+    /**
+     * Create a new {@link Session} for the user.
+     *
+     * @param user the {@link User} the session is being created for.
+     * @return a {@link Session} instance for the user.
+     * @throws IOException problem creating a session.
+     */
+    @Override
+    public Session create(User user) throws IOException {
+        SessionCreated sessionCreated = client.createNewSession(user.getEmail());
+        com.session.service.Session clientSession = client.getSessionByID(sessionCreated.getId());
+
+        if (clientSession == null) {
+            throw new IOException("client has failed to retrieve the session");
+        }
+
+        Session session = createZebedeeSession(clientSession);
+        return session;
+    }
+
+    /**
+     * Get a {@link Session} from the provided {@link HttpServletRequest}.
+     *
+     * @param request the {@link HttpServletRequest} to get the session for.
+     * @return a {@link Session} instance if once exists returns null otherwise.
+     * @throws IOException for any problem getting a sesison from the request.
+     */
+    @Override
+    public Session get(HttpServletRequest request) throws IOException {
+        String token = RequestUtils.getSessionId(request);
```

Comment:
Minor comment should rename this to `ID` or `sessionID` to be consistent with the rest of this class.